site/content/docs/0.0/layout/containers.md

```diff
@@ -8,4 +8,120 @@ aliases:
+<div class="container-2xl">100% wide until extra extra large breakpoint</div>
+<div class="container-3xl">100% wide until extra extra extra large breakpoint</div>
+```
+
+## Fluid containers
+
+Use `.container-fluid` for a full width container with minimum margins, spanning almost the entire width of the viewport.
+
+**`.container-fluid` is the container recommended for Orange sites in order to be compliant with the Orange brand.**
+
+```html
+<div class="container-fluid">
+  ...
+</div>
+```
+
+### Default max width
+
+This container is the default one to use, by default associated with the class `.container-max-width`. After breakpoint `2xl`, this class:
+- limits the container width to 1680px (including margins),
+- limits container margin to 80px each side,
+- limits gutters width to 32px.
+
+It is defined as follows:
+
+{{< scss-docs name="containers-max" file="scss/_containers.scss" >}}
+
+Comparison between `.container-fluid` and `.container-fluid` width `.container-max-width` can be seen in the following table:
+
+{{< bs-table "table" >}}
+|  | X-Large<div class="fw-normal">&ge;1320px</div> | 2X-Large<div class="fw-normal">&ge;1640px</div> | 3X-Large<div class="fw-normal">&ge;1880px</div> |
+| --- | --- | --- | --- | --- |
+| `.container-fluid` | <ul><li>Width `100% - (2 * 56px)`</li><li>Margin `2 * 56px`</li><li>Gutter `32px`</li></ul> | <ul><li>Width `100% - (2 * 80px)`</li><li>Margin `2 * 80px`</li><li>Gutter `32px`</li></ul> | <ul><li>Width `100% - (2 * 112px)`</li><li>Margin `2 * 112px`</li><li>Gutter `40px`</li></ul> |
+| `.container-fluid`<br>with `.container-max-width` | <ul><li>Width `100% - (2 * 56px)`</li><li>Margin `2 * 56px`</li><li>Gutter `32px`</li></ul> | <ul><li>Width `100% - (2 * 80px)`</li><li><b>Max-width `1520px`</b></li><li>Margin `2 * 80px`</li><li>Gutter `32px`</li></ul> | <ul><li><b>Width `1520px`</b></li><li><b>Margin `2 * 80px`</b></li><li><b>Gutter `32px`</b></li></ul> |
+{{< /bs-table >}}
+
+### Custom max width
+
+You can set a custom maximum width for the fluid container by overloading `$ouds-grid-container-max-width` before importing our OUDS scss files.
+
+This will affect all containers that have `.container-max-width` and will make sure that your layout does not exceed this value.
+When going up the breakpoint above `$ouds-grid-container-max-width` value, the container margins and gutter values are kept as they are.
+
+## CSS
+
+### Sass variables
+
+As shown above, OUDS Web generates a series of predefined container classes to help you build the layouts you desire. You may customize these predefined container classes by modifying the Sass map (found in `_variables.scss`) that powers them:
+
+{{< scss-docs name="container-max-widths" file="scss/_variables.scss" >}}
+
+For more information and examples on how to modify our Sass maps and variables, please refer to [the Sass section of the Grid documentation]({{< docsref "/layout/grid#css" >}}).
+
```

Comment:
Maybe this part should be changed if we want to stick to OB1's way to do it (having `.site-width` on a parent of `.container-fluid`). To see what we do for the start of this page: if we keep it like this, it starts with all the containers developers shouldn't use, it's confusing...